src/consts/origins.ts

```diff
@@ -0,0 +1 @@
+export const ALLOWED_ORIGINS = ["http://localhost:3001", "https://www.crossmint.io"];
```

Comment:
Are we also checking all this in the "backend"?

Reply:
This actually isnt something we would check on the crossmint.io backend side, cuz the backend side needs to be able to interact with any origin.

The reason we do it here is because we need to make sure the events received are in fact from crossmint